tests/attr/test_deconvolution.py

```diff
@@ -63,12 +66,20 @@ def test_simple_multi_input_conv_neuron_deconv(self):
             net, net.fc1, (3,), (inp, inp2), (ex_attr, ex_attr)
         )
 
-    def test_deconv_matching(self):
+    def test_deconv_matching(self) -> None:
         net = BasicModel_ConvNet_One_Conv()
         inp = 100.0 * torch.randn(1, 1, 4, 4)
         self._deconv_matching_assert(net, net.relu2, inp)
 
-    def _deconv_test_assert(self, model, test_input, expected, additional_input=None):
+    def _deconv_test_assert(
+        self,
+        model: Module,
+        test_input: TensorOrTupleOfTensors,
+        expected: Union[
+            Tuple[List[List[float]]], Tuple[List[List[float]], List[List[float]]]
```

Comment:
nit: This could be simpler by just having Tuple[List[List[float]], ...] for arbitrary length tuples.